CONTRIBUTORS.md

```diff
@@ -140,6 +140,14 @@ Tokyo Covid-19 response site contributors
 | Hidehiro Nagaoka | [@hidehiro98](https://github.com/hidehiro98) | [@hid9hiro](https://twitter.com/hid9hiro) | Coding, Translation |
 | y171 | [@totoro721](https://github.com/totoro721) | | Coding, Translation |
 | [hogesuke](https://hogesuke.hateblo.jp/) | [@hogesuke](https://github.com/hogesuke) | [@hogesuke_1](https://twitter.com/hogesuke_1) | Coding |
+| mcdmaster | [@mcdmaster](https://github.com/mcdmaster) | | Governance, Compliance, i18n |
+| Kazuya Atsumi | [@kazat](https://github.com/kazat) | | Coding |
+| hanapei00 | [@hanapei00](https://github.com/hanapei00) | | Coding |
+| Rikiya Ihara | [@magi1125](https://github.com/magi1125) | [@magi1125](https://twitter.com/magi1125) | Accessibility, Triage |
+| [Arthit Suriyawongkula](https://bact.cc) | [@bact](https://github.com/bact) | [@bact](https://twitter.com/bact) | Translation (Thai) |
```

Comment:
**Suriyawongkula** should be **Suriyawongkul**

(Line 147)
Thank you :)

Reply:
I fixed it. Thank you.